<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signwind</class>
 <widget class="QDialog" name="signwind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>E-Voting SignUp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-image: url(:/New folder/WhatsApp Image 2025-04-04 at 10.06.00 PM.jpeg);
    background-position: center;
    background-repeat: no-repeat;
}

/* Central container with glass morphism effect */
#centralwidget {
    background: rgba(255, 255, 255, 0.15);
    border-radius: 15px;
    border: 1px solid rgba(255, 255, 255, 0.2);
    padding: 30px;
    margin: 50px;
}

/* Title label */
#welcome_label {
    font: bold 28px &quot;Segoe UI&quot;, Arial;
    color: white;
    letter-spacing: 2px;
    text-transform: uppercase;
    qproperty-alignment: 'AlignCenter';
    margin-bottom: 30px;
}

/* Form labels */
QLabel {
    font: bold 16px &quot;Segoe UI&quot;;
    color: white;
    margin-bottom: 8px;
}

/* Input fields with modern styling */
QLineEdit {
    background: rgba(255, 255, 255, 0.25);
    border: 2px solid rgba(255, 255, 255, 0.4);
    border-radius: 8px;
    padding: 12px;
    color: white;
    font-size: 15px;
    margin-bottom: 15px;
    selection-background-color: #4facfe;
}

QLineEdit:focus {
    border-color: rgba(79, 172, 254, 0.8);
    background: rgba(255, 255, 255, 0.3);
}

/* Buttons with gradient and animation */
QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                              stop:0 #4facfe, stop:1 #00f2fe);
    color: white;
    font: bold 16px &quot;Segoe UI&quot;;
    border-radius: 8px;
    padding: 14px 24px;
    border: none;
    min-width: 120px;

}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                              stop:0 #00f2fe, stop:1 #4facfe);
}



/* Login container */
#login_container {
    background: rgba(255, 255, 255, 0.1);
    border-radius: 15px;
    padding: 30px;
    border: 1px solid rgba(255, 255, 255, 0.15);
}

/* Logo styling */
#logo_label {
    qproperty-alignment: 'AlignCenter';
    margin-bottom: 20px;
    qproperty-pixmap: url(:/icons/app_logo.png);
}
QDateEdit {
    background: rgba(255, 255, 255, 0.2);
    border: 1px solid rgba(255, 255, 255, 0.3);
    border-radius: 6px;
    color: white;
    font: 15px &quot;Segoe UI&quot;;
    padding: 8px 12px;
}

QDateEdit::drop-down {
    background: rgba(255, 255, 255, 0.1);
    border-radius: 0 4px 4px 0;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="2">
    <widget class="QPushButton" name="createacc_btn">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Sign Up To  Participate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Date Of Birth</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="pass_edit">
     <property name="placeholderText">
      <string>Password must contain letters and numbers</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="cnicedit">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Without Dashes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Cnic Number</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDateEdit" name="dateEdit">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="name_eidt">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>Enter your username</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
